<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TFCompositeModifier</class>
 <widget class="QWidget" name="TFCompositeModifier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>210</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>235</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>65</y>
     <width>225</width>
     <height>140</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAsNeeded</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>223</width>
      <height>138</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="manageWidget" native="true">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>35</y>
     <width>225</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="manageLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>Check period (ms):</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="delaySpin">
      <property name="minimumSize">
       <size>
        <width>55</width>
        <height>0</height>
       </size>
      </property>
      <property name="minimum">
       <number>20</number>
      </property>
      <property name="maximum">
       <number>100000</number>
      </property>
      <property name="singleStep">
       <number>10</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="manageButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>5</y>
     <width>115</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>115</width>
     <height>25</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>115</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>Manage Composition</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
